lib/grandpa/finalization_test.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
```

Comment:
Many of these tests sleep and take several seconds, please move these to an integration test file and add a lib/grandpa integ test line to the matrix in the github workflow